osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/DlmsObjectConfig.java

```diff
@@ -50,4 +52,20 @@ boolean contains(Protocol protocol) {
                 .findAny();
         // @formatter:on
     }
+
+    public ObisCode getObisForObject(final DlmsObjectType type) throws
```

Comment:
public method missing Javadoc

Reply:
Add Javadoc

Reply:
Javadoc is not mandatory anywhere. I usually only add it on interfaces with multiple implementations. More Javadoc is more maintenance and probably more out-of-date information. Try to write self-documenting code and avoid comments/Javadoc where possible.

Reply:
if we agree on that it is fine. Also last project "No Javadoc on public method" failed the Sonar QA gate. So I am used to doing this.

Reply:
Writing down comments does NOT save time, ever. Everything you write down costs time to formulate, maintain and read (n times). Not to mention the confusion is can cause when comments no longer match reality. Comments are per definition redundant additions, but might be needed to:

- explain intent
- clarify code.
- warn of consequences

See also; https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29